bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -56,11 +56,7 @@ class LayoutDOMView extends BokehView
 
   bind_bokeh_events: () ->
     @listenTo(@model, 'change', @render)
-
-    if @model.sizing_mode == 'fixed'
-      @listenToOnce(@model.document.solver(), 'resize', @render)
-    else
-      @listenTo(@model.document.solver(), 'resize', @render)
+    @listenTo(@model.document.solver(), 'resize', () => @render())
```

Comment:
@bokeh/dev, make sure to use `() => @fn()` instead of just `@fn`, if `fn` is not an event handler. `render` is not. This way you make sure that no random arguments will be passed to `fn`. This makes code understanding and refactoring easier. Also, not that important, but `printf`-style debugging is much more convenient this way.